### PR TITLE
Capability to write to Serpent

### PR DESCRIPTION
**Pull request by alexvalentine94**
Initial effort at also writing Serpent input files

Some testing success for simple models but needs further tests and edge cases.

Hopefully useful for this great tool :) 

Reverse reading of Serpent -> CAD in progress but will raise separate PR.

### CODE REVIEW

File: src/GEOUNED/Write/Functions.py [hunk 1]
```diff
@@ -162,7 +186,8 @@ def writeSequenceOMCPY(Seq,prefix='S'):
      return line
 
 def MCNPSurface(id,Type,surf):
-    MCNP_def = ''
+    Serpent_def = ''
```
Comment (jpcatalanUNED) [inline src/GEOUNED/Write/Functions.py:189]: This change is wrong. Should be 
MCNP_def = ''

**Review by jpcatalanUNED — APPROVED** [review]
The changes proposed will be accepted. There is a little bug in Functions.py line 189, we will resolve it.

**Comment by jpcatalanUNED** [comment]
@alexvalentine94 thank you for your contribution!

File: src/GEOUNED/Write/Functions.py (hunk 1 above)
Comment (alexvalentine94) [reply src/GEOUNED/Write/Functions.py:189]: good spot!
